packages/components/ng-ovh-uirouter-layout/rollup.config.js

```diff
@@ -0,0 +1,25 @@
+import path from 'path';
+import rollupConfig from '@ovh-ux/component-rollup-config';
+
+const config = rollupConfig({
+  input: './src/index.js',
+}, {
+  lessTildeImporter: {
+    paths: [
+      path.resolve(__dirname, 'node_modules'),
+      path.resolve(__dirname, '../../../../node_modules'),
+    ],
+  },
+});
+
+export default [
+  config.cjs(),
+  config.umd({
```

Comment:
Don't you need to also  add 
```suggestion
  config.es()
```
?

Reply:
👍